talentlink-connector-product/README.md

```diff
@@ -0,0 +1,37 @@
+# TalentLink Connector
+Axon Ivy�s TalentLink connector helps you accelerate your process automation initiatives by integrating TalenLink with your process application in the shortest possible time.
```

Comment:
Does "TalenLink" have a public landing page from the vendor? Personally I don't know TalentLink ... so it would help to see what this connector is trying to connect.